Automattic/wp-calypso
# Update plans-visual.svg

This change updates the image file used with the JITM modal component, as the current version is now outdated.

#### Testing instructions

Apply the appropriate tag to your WordPress.com account, and load any free site in Calypso. Confirm that the image file used within the modal component is the updated version.
